app/commands/login/download_and_create_pair.rb

```diff
@@ -50,6 +50,7 @@ def build_user(pair, number, pair_data)
         city: pair_data[:"city#{number}"],
         phone: pair_data[:"phone#{number}"],
         email: pair_data[:"email#{number}"]
+        #facebook: pair_data[:"facebook#{number}"] TODO: uncomment after website api update
```

Comment:
to jest trochę dead code. W sensie czaję że sobie to odkomentujesz potem ale skoro tak to czemu po prostu nie zmergujesz już odkomentowanego kodu potem